### PR TITLE
Fix build kf4cic fail in USE_TRUSTY disable case

### PR DESCRIPTION
**Pull request by GraspEmpty**
Tracked-On: OAM-86757
Signed-off-by: sunxunou <xunoux.sun@intel.com>

### CODE REVIEW

**Comment by GraspEmpty** [comment]
EB master-engineering/builds/94910 test pass

File: kf4aic.c [hunk 1]
```diff
@@ -186,8 +186,8 @@ EFI_STATUS start_systemd_boot(EFI_HANDLE image_handle)
 	return ret;
 }
 
-#ifdef USE_TRUSTY
 struct rot_data_t g_rot_data = {0};
```
Comment (jeremy-compostella) [inline kf4aic.c]: In general having such a global variable is a poor software architecture design.  Why can't you define an accessor interface, or have it at a local variable and then submit it to the right module ?

Comment (jwu55) [reply kf4aic.c]: Jeremy, i see your point. g_rot_data define here is just same with what we did in kernelfliner.c and kf4abl.c before. I agree global variable is not a good idea and at least it should be static one.  Xunou, before fix it, we need refine keymaster.c  which was using this g_rot_data

Comment (jwu55) [reply kf4aic.c]: Jeremy, please ignore this patch, i have created  to address global variable issue your mentioned
https://github.com/intel/kernelflinger/pull/37

**Pull request closed without merging**